src/util.cpp

```diff
@@ -653,15 +653,15 @@ bool RenameOver(fs::path src, fs::path dest)
 }
 
 /**
- * Ignores exceptions thrown by Boost's create_directory if the requested directory exists.
+ * Ignores exceptions thrown by Boost's create_directories if the requested directory exists.
```

Comment:
But it is using Boost.

`namespace fs = boost::filesystem;`.

I fixed the comment at the end of the function.